examples/common.cpp

```diff
@@ -274,6 +274,24 @@ bool gpt_params_parse(int argc, char ** argv, gpt_params & params) {
                 break;
             }
             params.cfg_negative_prompt = argv[i];
+        } else if (arg == "--cfg-negative-prompt-file") {
+            if (++i >= argc) {
+                invalid_param = true;
+                break;
+            }
+            std::ifstream file(argv[i]);
+            if (!file) {
+                fprintf(stderr, "error: failed to open file '%s'\n", argv[i]);
+                invalid_param = true;
+                break;
+            }
+            std::copy(
+                std::istreambuf_iterator<char>(file),
+                std::istreambuf_iterator<char>(),
+                back_inserter(params.cfg_negative_prompt));
```

Comment:
Could you make this into one line like it is in `--file`?

Reply:
Sure. I just used the style from the `--grammar-file` section but I can put it on one line if you prefer that.